tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -534,6 +534,12 @@ function updateNpmScripts(tree: Tree, options: NormalizedSchema) {
 
     Object.assign(json.scripts, scripts);
 
+    const isNodePackage = options.projectConfig.tags?.some(tag => tag === 'platform:node');
```

Comment:
Should we only apply the `start` scripts for `platform:web` instead?

Reply:
I'd keep it as is - in future we should get rid of `start` completely as it can mean various things for various environment and rather be specific.

example: 
- I'm going to run storybook -> run `storybook` command
- Im going to run tests  -> run `test` command (we have this already)
- Im going to run e2e -> run `e2e` command (we have this partially - web server needs to be run manually )
- etc